Tiqr/Resources/General/Tiqr-Info.plist

```diff
@@ -52,6 +52,11 @@
 	<string>1.0</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSAllowsArbitraryLoads</key>
+		<true/>
```

Comment:
Yes, App Transport Security is actually a Good Thing. Especially during enrolment, where the client shares a secret with the server, it is pivotal that communication is protected. Ideally, Perfect Forward Secrecy is used to keep eavesdroppers from obtaining the secret in case of a future key compromise.